presto-main/src/main/java/com/facebook/presto/sql/gen/PageFunctionCompiler.java

```diff
@@ -157,14 +157,19 @@ public CacheStatsMBean getFilterCache()
     }
 
     public Supplier<PageProjection> compileProjection(SqlFunctionProperties sqlFunctionProperties, RowExpression projection, Optional<String> classNameSuffix)
```

Comment:
This (and any other place we hard code to false) should have the `@VisibleForTesting` annotation

Reply:
Another place that should have the annotation still

Reply:
Done